.github/copilot-instructions.md

```
@@ -1,571 +1,34 @@
-  x < 100 
-}
-```
-
-Conditions are required to be defined at the end of the model (after the type definitions), and are instantiated using conditional relationship tuples.
-
-### **Leveraging Usersets for Group-Based Access Control**
-
-A userset represents a set or collection of users, denoted by object#relation (e.g., `company:xy#employee`). Usersets are fundamental for assigning permissions to groups, reducing tuple count and providing flexibility for bulk access management. They can be used in direct relationship type restrictions, such as:
+For the complete set of rules, install the skill:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)